master/getting-started/mesos/installation/docker.md

```diff
@@ -5,65 +5,92 @@ title: Installing Calico for the Docker Containerizer
 This document provides the commands to download and run Calico
 for use with the Docker Containerizer in Mesos.
 
-## Prerequisite: Docker
-Calico networks Docker tasks for Mesos with its libnetwork plugin. In order to
-run the libnetwork plugin, the Docker daemon on each agent must be configured
-with a cluster store.
+## Prerequisites
 
-If using etcd as a cluster store, for example, run the Docker daemon with the
-following additional parameter:
+#### 1. etcd
 
-```shell
-    --cluster-store=etcd://<ETCD HOST>:<PORT>
+Calico uses etcd as its datastore. Ensure you have an instance of etcd running,
+and that it is accessible from all Agents in your cluster.
+
+To maximize availability, we recommend running an etcd cluster across your Mesos masters.
+See [etcd's clustering guide](https://coreos.com/os/docs/latest/cluster-architectures.html)
+for more information.
+
+For testing, you can run a single instance of etcd using docker on your Master
+with the following command, ensuring you've correctly set `$ETCD_IP`:
+```
+docker run --detach \
+	--net=host \
+	--name etcd quay.io/coreos/etcd:v2.0.11 \
+	--advertise-client-urls "http://$ETCD_IP:2379" \
+	--listen-client-urls "http://$ETCD_IP:2379,http://127.0.0.1:2379"
 ```
 
-Replacing `<ETCD HOST>:<PORT>` with the appropriate `hostname:port`
-for your etcd cluster.
+#### 2. Docker Configured with Cluster Store
 
-## Install and Run Calico
-It is very easy to install Calico to use with the
-Docker Containerizer.
+Under the covers, Calico networks Docker tasks for Mesos with its Docker CNM plugin.
+Multihost Networking in Docker requires that
+each Agent's Docker daemon must be configured with a cluster store.
 
-1. On each Mesos Agents, download the `calicoctl` command-line tool:
+Though Docker's configured cluster-store does not have to be the same as Calico's,
+for simplicity, users can configure Docker to use the same by setting the
+following flag in `/etc/docker/daemon.json`:
```

Comment:
Do we stipulate a minimum Docker version?   The daemon.json i believe is only valid from 1.10 - prior to 1.10 you have to specify this as the command options.